src/lib/cozy-client/reducer.js

```diff
@@ -314,15 +314,15 @@ export const trashFile = (file, actionOptions = {}) => ({
 
 export const addReferencedFiles = (doc, ids) => ({
   type: ADD_REFERENCED_FILES,
-  collection: `${doc.type}/${doc.id}#files`,
+  collection: `${doc._type}/${doc._id}#files`,
```

Comment:
I guess I trust you on this one.

Reply:
Previously, `cozy-client` relied on the `type` property, which is used by some doctypes like `io.cozy.files`. So now we rely on the `_type` property.